lib/greet.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = function(name){
+  if (!name)
```

Comment:
Read the Feature Tasks section again. This function needs to check if the argument is a string. This doesn't do that, which means if I invoke it by passing ```7```, I would get back ```Hello 7``` which shouldn't happen.